pages/linux/curlie.md

```diff
@@ -0,0 +1,20 @@
+# curlie
+
+> Curlie is a frontend to curl that adds the ease of use of httpie.
+> More information: <https://github.com/rs/curlie>.
+
+- Send a GET request:
+
+`curlie httpbin.org/get`
```

Comment:
can we add our token syntax to these links please.

Reply:
Okay, you should also check `xh` page as I have referred it. Feel free to add the suggestions.